### ISSUE TITLE
`prefer-const-border-radius` for entire file

### ISSUE THREAD

**Issue by FMorschel** [dart-code-checker/dart-code-metrics#1130]
**Shouldn't there be a `[prefer-const-border-radius](https://dartcodemetrics.dev/docs/rules/flutter/prefer-const-border-radius)` quick-fix for the entire file?**

Since mostly all quick-fixes have a quick-fix for the entire file, shouldn't this one have this function too?

**Comment by incendial** [comment]
@FMorschel DCM does not yet have fixes for more than 1 issue at a time. Do what other quick fixes do you refer?

**Comment by FMorschel** [comment]
Oh, I see, I thought there were quick-fixes for more than 1 issue at a time just like Dart/Flutter lints. Are there any thoughts on doing that?

**Comment by incendial** [comment]
Yeah, it's actually already implemented, but will become available later this month, stay tuned 🙂 

<img width="351" alt="Screenshot 2023-01-02 at 21 45 31" src="https://user-images.githubusercontent.com/21116742/210264466-ef7832b6-9b81-4b06-90d8-4a20562289de.png">

**Comment by incendial** [comment]
Available in DCM for Teams version https://dcm.dev/for-teams
